src/Loadbalancers/Entities/Header.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace UKFast\SDK\Loadbalancers\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $type
+ * @property string $subtype
```

Comment:
```suggestion
 * @property string $subtype
 * @property string $header
```
Just a missing comment